Sources/AppcuesKit/Data/Networking/Authentication.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Authorization.swift
+//  AppcuesKit
+//
+//  Created by James Ellis on 2/14/23.
+//  Copyright © 2023 Appcues. All rights reserved.
+//
+
+import Foundation
+
+internal enum Authorization {
+    case bearer(String)
```

Comment:
Some speculation here to encapsulate possibly other future auth types inside this type. In reality, we could probably flatten and remove, don't feel strongly.